### ISSUE TITLE
make it more obvious who is who

### ISSUE THREAD

**Issue by CaptainSlumbo** [CrashpointF13/CrashpointF13#73]
I find it hard sometimes when there's a lot of people on my screen to figure out who is who is who, and I find it even harder to remember that. this is especially true when the server is highly populated and you are dealing with factions. I think other server's already tackled this problem correctly.

My suggestion is to add a rank prefix to the main factions, NCR/BOS/LEGION
example: a Decanus named "sillius italicus" will appear in chat as just Sillius Italicus, this means that if I want to know his rank I need to shift click him and view his attire and see if I can look at his ID a lot of new users just dont know this/dont care. so adding a name prefix would help with that

I think this is also important for NCR, especially their radio. I think if people see the rank of whoever is giving the orders over the radio it will prevent a lot of command-chain related fuck ups, and realistically you'd recognize all the voices of all the troopers you interact with.

tl;dr
- display a rank prefix to faction-related roles in chat and in radio

**Comment by imw-Oculus** [comment]
I feel like the uniforms make faction allegiance fairly obvious. It's not really that much of a hassle to literally shift-click to find out someone's rank. If a faction commander is doing a good job, it will be obvious who you are reporting to - but people should maintain the freedom to do their jobs poorly.

**Comment by CaptainSlumbo** [comment]
It has nothing to do with differentiating factions at all as you said yourself the uniforms make it fairly obvious, it's about differentiating people within the same faction easily.

**Comment by imw-Oculus** [comment]
I can definitely see where you're coming from, I'm just always hesitant to make yet another sacrifice on the altar of vidya game mechanics >>> realism. I think having proper, good commanders who ensure that everyone knows who is in charge of who makes this a rather needless change - and gives them the opportunity to be bad at their jobs.

**Comment by CaptainSlumbo** [comment]
I don't think leadership is as easy as, "just lead 4head", and also assuming things of other people is a very bad and toxic habit, people still need to learn the game and they can only learn if you are nice and play along with them. otherwise you can only blame yourself for playing a social game and not being sociable.

**Comment by ProbablyCarl** [comment]
Introduction of Runechat has corrected many of these issues.
